### ISSUE TITLE
Exception happens when I'm trying to open generated pdf using iTextSharp library 

### ISSUE THREAD

**Issue by shashankSkydale** [stchan/PdfScribe#20]
Hi,
After pdf generation by ghostscript method (GhostScript64.CallAPI(ghostScriptArguments) 
I'm trying to open pdf using iTextSharp library to read as text but exception is occurring which is caught under catch (IOException ioEx).

Below is my sample code snippet

```
GhostScript64.CallAPI(ghostScriptArguments);
DisplayPdf(outputFilename);
String pdfContent = ReadPDF(outputFilename);
...

static String ReadPDF(String outputFilename)
{
    StringBuilder text = new StringBuilder();
     
    using (PdfReader reader = new PdfReader(outputFilename))
    {
        for (int i = 1; i <= reader.NumberOfPages; i++)
        {
            text.Append(PdfTextExtractor.GetTextFromPage(reader, i));
        }
        reader.Close();
    }
    return text.ToString();
}

```

**Comment by shashankSkydale** [comment]
Please assist

**Comment by stchan** [comment]
I am not the author, or a maintainer of the itextpdf product.

**Comment by shashankSkydale** [comment]
> 
> 
> I am not the author, or a maintainer of the itextpdf product.

Hi stchan,
I understand and completely agree with you that you are not the author of itextpdf product but I'm not able to understand why opening pdf is resulting in exception(**Could not create the output file. <filename> is in use**)

I've tried itextpdf library in another project and it works completely fine.
Also I have used below library but still the same issue.

`https://www.sautinsoft.com/products/pdf-focus/how-to-extract-text-from-pdf-in-dotnet-aspnet-csharp-vb.php`
